index.adoc

```diff
@@ -38,6 +38,10 @@ This section explains how to upgrade your current Taiga to a newer version
 
 This section shows how to keep your link:upgrades-6to6.html[taiga-docker instance up to date] with the new releases.
 
+=== Backup and Restore Taiga6 docker
```

Comment:
This should be a "first level" section, similar to `== Upgrading Guide`. Its title doesn't need to be coupled to docker, because we could expand the documentation for instances from source code. So my suggestion is that this secion should be somethink like:

```
== Backup and Restore Taiga

How to link:backup-and-restore.html#docker[backupd and restore] Taiga6 docker.

```

the `#docker` part means that the document may contain both explanations (so far we only have the docker one, step by step).